daemon/daemon.c

```diff
@@ -487,12 +492,7 @@ int become_daemon(int dont_fork, const char *user)
         else debug(D_SYSTEM, "Successfully became user '%s'.", user);
     }
     else {
-        create_needed_dir(netdata_configured_cache_dir, getuid(), getgid());
-        create_needed_dir(netdata_configured_varlib_dir, getuid(), getgid());
-        create_needed_dir(netdata_configured_lock_dir, getuid(), getgid());
-        create_needed_dir(claimingdirectory, getuid(), getgid());
-
-        clean_directory(netdata_configured_lock_dir);
+        prepare_required_directories(getuid(), getgid());
```

Comment:
I can see in this source code a potential new reviewer. :thinking:

Reply:
ping @stelfrag

Reply:
> I can see in this source code a potential new reviewer.

Perhaps, though only for the comparatively less complicated stuff.